src/app/pages/storage/volumes/volume-import-wizard/volume-import-wizard.component.ts

```diff
@@ -348,6 +350,32 @@ export class VolumeImportWizardComponent implements WizardConfiguration {
       dialogRef.componentInstance.submit();
```

Comment:
There is another if above:
```
if (value.encrypted) {
```
Do you know what it's for?

Reply:
I think it's to check if the pool being imported is encrypted itself and needs a key or passphrase along with form submission.

Reply:
It's for when there are encrypted disks involved. According to @sonicaj `12.0-stable` is a good example to follow and it does the same thing.

Reply:
One of the two possibilities seem to exist. Either this is an old code and is unused (as I couldn't seem to hit that code even when using encrypted pools) or the code is used and it may need to be rewritten to add support for unlocking locked datasets.

Reply:
Yes. Upon further investigation, it looks like Scale is missing this step from Core that works on decrypting disks
![image](https://user-images.githubusercontent.com/24204846/141377497-7bb218f1-d6ca-425f-ace3-05b460717718.png)

Reply:
So, it can be removed. Just need @sonicaj to confirm.

Reply:
This page has legacy code all over that changes the form based on whether we're working with Scale or Core. So, upon confirmation, all of that can be removed since this is a Scale branch now and checking for those is not needed.

Reply:
@RehanY147 decrypting disks code can be removed from SCALE as that is specific to CORE. We would only in this case want to decrypt datasets after importing a pool.